package.json

```diff
@@ -125,6 +125,166 @@
+                                            "template"
+                                        ]
+                                    }
+                                },
+                                "additionalProperties": false,
+                                "required": [
+                                    "target"
+                                ],
+                                "type": "object"
+                            },
+                            "lambda": {
+                                "title": "Lambda Properties",
+                                "description": "%AWS.configuration.description.awssam.debug.lambda%",
+                                "properties": {
+                                    "environmentVariables": {
+                                        "description": "%AWS.configuration.description.awssam.debug.envvars%",
+                                        "$ref": "#/definitions/jsonObject"
+                                    },
+                                    "event": {
+                                        "description": "%AWS.configuration.description.awssam.debug.event%",
+                                        "properties": {
+                                            "json": {
+                                                "$ref": "#/definitions/jsonObject"
+                                            },
+                                            "path": {
+                                                "description": "%AWS.configuration.description.awssam.debug.event.path%",
+                                                "type": "string"
+                                            }
+                                        },
+                                        "additionalProperties": false,
+                                        "type": "object"
+                                    },
+                                    "memoryMb": {
+                                        "description": "%AWS.configuration.description.awssam.debug.memoryMb%",
+                                        "type": "number"
+                                    },
+                                    "runtime": {
+                                        "description": "%AWS.configuration.description.awssam.debug.runtime%",
+                                        "type": "string"
+                                    },
+                                    "timeoutSec": {
+                                        "description": "%AWS.configuration.description.awssam.debug.timeout%",
+                                        "type": "number"
+                                    }
+                                },
+                                "additionalProperties": false,
+                                "type": "object"
+                            },
+                            "sam": {
+                                "title": "SAM CLI Properties",
```

Comment:
nit: `Sam Cli Properties`? Seems like that would generated `SamCliProperties` instead of `SAMCLIProperties`

Reply:
This is shown in the tooltip - I want to optimize for the customer UX rather than the generated class names.

Reply:
Ah. Interesting choice taken by `json-schema-to-typescript` to infer an interface from the title. Looks like it just inlines the type if you don't specify the title.